rir/src/interpreter/interp.cpp

```diff
@@ -3057,6 +3144,9 @@ SEXP evalRirCode(Code* c, InterpreterInstance* ctx, SEXP* env,
             synthesizeFrames->pop_front();
             SEXP res = ostack_pop(ctx);
             SEXP e = ostack_pop(ctx);
+            if (LazyEnvironment::check(e)) {
+                e = (LazyEnvironment::unpack(e))->create();
+            }
```

Comment:
why don't we need to update the stack here?

Reply:
We need and we are doing it.